parm/config/gfs/config.resources

```diff
@@ -357,12 +357,12 @@ case ${step} in
 
   "ocnanalbmat")
     npes=16
-    case ${CASE} in
-      "C384") npes=480;;
-      "C96")  npes=16;;
-      "C48")  npes=16;;
+    case ${OCNRES} in
```

Comment:
I don't think 5-deg case needs 480 pes and the 0.25-deg case runs on 16.

Reply:
now unreversed